### PR TITLE
[FR] Update `Beatmap/Approach_rate`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Approach_rate/fr.md [hunk 1]
```diff
@@ -15,6 +13,8 @@ L'**Approach Rate** (abrégé en ***AR***) est un paramètre de difficulté d'un
 
 Les valeurs de l'AR vont de 0 à 10. Un approach rate plus élevé signifie que les objets seront affichés pendant une période plus courte, ce qui laisse moins de temps au joueur pour réagir. En revanche, des approach rates plus faibles permettent plus de temps de réaction, mais peuvent entraîner l'apparition d'un nombre excessif d'objets à l'écran en même temps.
 
+Dans [osu!taiko](/wiki/Game_mode/osu!taiko) et [osu!mania](/wiki/Game_mode/osu!mania), le réglage de l'approach rate n'a aucun effet. La vitesse de défilement dans les deux modes est contrôlée par le [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), qui dépend du [BPM](/wiki/Music_theory/Tempo) et le multiplicateur de slider velocity. De plus, dans osu!mania, il peut être personnalisé [par le joueur lui-même](/wiki/Game_mode/osu!mania#changement-de-vitesse).
```
Comment (Pisapou) [inline wiki/Beatmap/Approach_rate/fr.md:16]:
```suggestion
Dans [osu!taiko](/wiki/Game_mode/osu!taiko) et [osu!mania](/wiki/Game_mode/osu!mania), le réglage de l'approach rate n'a aucun effet. La vitesse de défilement dans les deux modes est contrôlée par le [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), qui dépend du [BPM](/wiki/Music_theory/Tempo) et du multiplicateur de slider velocity. De plus, dans osu!mania, elle peut être personnalisé [par le joueur lui-même](/wiki/Game_mode/osu!mania#changement-de-vitesse).
```

File: wiki/Beatmap/Approach_rate/fr.md [hunk 2]
```diff
@@ -49,8 +53,4 @@ Il existe quatre mods qui modifient l'approach rate lorsqu'ils sont activés :
...
 - [Half Time](/wiki/Game_modifier/Half_Time) : La valeur de l'AR n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse de jeu, les objets restent à l'écran 33 % plus longtemps.
 
-Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'AR, la différence de vitesse y entraîne un changement notable. Les AR de HT/DT sont généralement désignés en fonction de leur valeur perçue. Par exemple, "AR8+DT" peut également être écrit "AR9.6".
-
-## osu!taiko et osu!mania
-
-L'approach rate est visible lors de l'affichage des informations d'une beatmap d'[osu!taiko](/wiki/Game_mode/osu!taiko) ou d'[osu!mania](/wiki/Game_mode/osu!mania), mais elle n'affecte pas la jouabilité. La vitesse de défilement de chaque mode est déterminée par d'autres facteurs.
+Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'AR, la différence de vitesse y entraîne un changement notable. Les AR de HT/DT sont généralement désignés en fonction de leur valeur perçue. Par exemple, "AR 8 + DT" peut également être écrit "AR 9.6".
```
Comment (Ganondorf-1) [inline wiki/Beatmap/Approach_rate/fr.md:56]:
```suggestion
Bien que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'AR, la différence de vitesse y entraîne un changement notable. Les AR de HT/DT sont généralement désignés en fonction de leur valeur perçue. Par exemple, "AR 8 + DT" peut également être écrit "AR 9.6".
```

File: wiki/Beatmap/Approach_rate/fr.md [hunk 3]
```diff
@@ -3,8 +3,6 @@ tags:
   - approach time
   - AR
   - reading
```
Comment (Ganondorf-1) [inline wiki/Beatmap/Approach_rate/fr.md:5]: Maybe we should add `temps d'approche` here ? It's the French translation of `Approach time` on the stable client

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Approach_rate/fr.md [hunk 4]
```diff
@@ -15,6 +13,8 @@ L'**Approach Rate** (abrégé en ***AR***) est un paramètre de difficulté d'un
 
 Les valeurs de l'AR vont de 0 à 10. Un approach rate plus élevé signifie que les objets seront affichés pendant une période plus courte, ce qui laisse moins de temps au joueur pour réagir. En revanche, des approach rates plus faibles permettent plus de temps de réaction, mais peuvent entraîner l'apparition d'un nombre excessif d'objets à l'écran en même temps.
 
+Dans [osu!taiko](/wiki/Game_mode/osu!taiko) et [osu!mania](/wiki/Game_mode/osu!mania), le réglage de l'approach rate n'a aucun effet. La vitesse de défilement dans les deux modes est contrôlée par le [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), qui dépend du [BPM](/wiki/Music_theory/Tempo) et du multiplicateur de slider velocity. De plus, dans osu!mania, elle peut être personnalisé [par le joueur lui-même](/wiki/Game_mode/osu!mania#changement-de-vitesse).
```
Comment (Ganondorf-1) [inline wiki/Beatmap/Approach_rate/fr.md:16]:
```suggestion
Dans [osu!taiko](/wiki/Game_mode/osu!taiko) et [osu!mania](/wiki/Game_mode/osu!mania), le réglage de l'approach rate n'a aucun effet. La vitesse de défilement dans les deux modes est contrôlée par le [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), qui dépend du [BPM](/wiki/Music_theory/Tempo) et du multiplicateur de slider velocity. De plus, dans osu!mania, elle peut être personnalisée [par le joueur lui-même](/wiki/Game_mode/osu!mania#changement-de-vitesse).
```

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**